src/app/problem/edit/testCase/TestCases.js

```diff
@@ -0,0 +1,101 @@
+import {TestCaseSubtitleLine} from "../../../commons/titles/TitleLine";
+import React from "react";
+import './TestCase.css';
+import {InlineInputBox} from "../InlineInputBox";
+import {UploadFileButton} from "../UploadFileButton";
+import {EditorButton} from "../EditorButton";
+
+function TestCases({testCasesItems, removeTestCase}) {
+    return testCasesItems.map((testCaseId) => (
+        <div className="test-case-box-out" key={testCaseId}>
+            <div >
+                <span className="test-case-ame">Test Case Name</span>
+                <input className="text-case-name-input" />
+            </div>
+            <div className="test-case-box-in columns">
+                <div className="column is-two-fifths">
+                    <TestCaseSubtitleLine title="Limits"/>
+                    <InlineInputBox title="Time Limit" fontSize={14}/>
+                    <InlineInputBox title="Memory Limit" fontSize={14}/>
+                    <InlineInputBox title="Output Limit" fontSize={14}/>
+
+                    <TestCaseSubtitleLine title="Grade" fontSize={15}/>
+                    <InlineInputBox title="Grade" fontSize={14}/>
+                </div>
+                <div className="column">
+                    <TestCaseSubtitleLine title="Standard In"/>
+                    <UploadFileButton
+                        title={"Standard In +"}
+                        width={110} height={28}
+                        buttonColor={"#F2B311"}
+                        borderRadius={10}
+                        fontWeight={600}
+                        fontSize={12}
+                        lineHeight={18}
+                        fontColor={"white"}
+                        display={"flex"}
+                    />
```

Comment:
You can move the same css to `UploadFileButton.css` or use the className to change the css style in different situations.